packages/webview_flutter/example/android/app/src/androidTest/java/io/flutter/plugins/webviewflutterexample/WebViewTest.java

```diff
@@ -0,0 +1,16 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.webviewflutterexample;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class WebViewTest {
+  @Test
+  public void placeHolderTest() {
+    assertEquals(1, 1);
```

Comment:
Could you do something like instantiate the plugin class here, so that we know it's linking in correctly?

Reply:
Done